packages/react-core/src/components/Badge/examples/BadgeUnread.tsx

```diff
@@ -3,6 +3,17 @@ import { Badge } from '@patternfly/react-core';
 
 export const BadgeUnread: React.FunctionComponent = () => (
   <React.Fragment>
-    <Badge key={1}>7</Badge> <Badge key={2}>24</Badge> <Badge key={3}>240</Badge> <Badge key={4}>999+</Badge>
+    <Badge key={1} screenReaderText="7 Unread Messages">
+      7
+    </Badge>
+    <Badge key={2} screenReaderText="24 Unread Messages">
+      24
+    </Badge>
+    <Badge key={3} screenReaderText="240 Unread Messages">
+      240
+    </Badge>
+    <Badge key={4} screenReaderText="More Than 999 Unread Messages">
```

Comment:
Should be the last change! For the screenReaderText, we can omit the number since that will already be announced by assistive tech.

Right now VoiceOver for example announces "7 7 unread messages". If we update the screenReaderText to just "Unread messages", it should provide the expected output

Reply:
Ooh got it now, sorry for taking so much of your time, going to correct that now.

Reply:
No need to apologize! It's always worth the time to review things (especially when we can sneak in a mini accessibility learning lesson 😆 )

Reply:
It was perfect for me for learning sth. on a11y, I wasn't edited or created anything for a11y before.

Reply:
You did exception-a11y well!  💪🏼😎